core/sql/regress/executor/EXPECTED130

```diff
@@ -124,7 +134,16 @@ inserted row13
 
 --- 3 row(s) selected.
 >>
->>
+>>values (filetolob('./myfile.txt'));
+
+*** ERROR[8442] Unable to access ExpLOB interface. Call to ExpLOB returned error LOB_PTR_ERROR(532). Error detail 0.
```

Comment:
That sounds better. Now a user knows what to do. They need to add some function that converts the LOB to some other datatype (or processes it somehow to a more palatable form). Thanks for considering this.